### PR TITLE
Update Shebangs and daily.sh for FreeBSD compatibility

### PR DESCRIPTION
**Pull request by Pricetx**
Updated the shebangs on any files that did not have flags set to use /usr/bin/env, to provide support for FreeBSD in addition to Linux.
Updated daily.sh to support FreeBSD (su has different args, replaced with sudo when not on linux)

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: daily.sh [hunk 1]
```diff
@@ -113,7 +113,11 @@ main () {
         # only try to su if we are root (or sudo)
         if [[ "$EUID" -eq 0 ]]; then
             echo "Re-running ${DAILY_SCRIPT} as ${LIBRENMS_USER} user"
-            su -l "$LIBRENMS_USER" -c "$DAILY_SCRIPT $@"
+            if [[ "$(uname)" == "Linux" ]]; then
+                su -l "$LIBRENMS_USER" -c "$DAILY_SCRIPT $@"
+            else
+                sudo -u "$LIBRENMS_USER" "$DAILY_SCRIPT"
```
Comment (murrant) [inline daily.sh]: You are missing the arguments here `$@`

Also, we could probably use sudo for both Linux and FreeBSD...

Comment (Pricetx) [inline daily.sh]: Thanks, I had initially left it off due to it causing the script to break, but later realised it was just a quoting issue. I had intended to add it back but apparently forgot to. I will fix that now.

Regarding sudo on Linux, I'm perfectly fine to do so, but I didn't want to unnecessarily impose the additional dependency on Linux users if I didn't have to. If you're happy for everything to use sudo then i'll go ahead and remove the if statement altogether.

Comment (murrant) [inline daily.sh]: I think sudo is very standard.

**Comment by Pricetx** [comment]
I have updated the pull request with the following changes:
- Added missing arguments to sudo command
- Removed su command so all platforms use sudo

**Comment by laf** [comment]
lgtm

**Pull request merged**